### PR TITLE
Add "spring.cloud.azure.active-directory.enabled" in additional-spring-configuration-metadata.json

### PR DESCRIPTION
**Pull request by chenrujun**
As title

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @backwind1233 , @saragluna , please help to review this PR when you have time.

File: sdk/spring/spring-cloud-azure-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json [hunk 1]
```diff
@@ -932,6 +932,12 @@
         "replacement": "spring.cloud.stream.servicebus.bindings"
       }
     },
+    {
+      "name": "spring.cloud.azure.active-directory.enabled",
+      "type": "java.lang.Boolean",
+      "description": "Whether enable Azure Active Directory related auto configuration.",
+      "defaultValue": false
+    },
```
Comment (saragluna) [inline sdk/spring/spring-cloud-azure-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json:940]: do we need such thing for aad b2c?

**Review by saragluna — APPROVED** [review]

File: sdk/spring/spring-cloud-azure-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json (hunk 1 above)
Comment (saragluna) [reply sdk/spring/spring-cloud-azure-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json:940]: Whether to enable

Comment (chenrujun) [reply sdk/spring/spring-cloud-azure-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json]: Updated.

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**